### ISSUE TITLE
Define a set of plasma regimes for tests

### ISSUE THREAD

**Issue by lemmatum** [PlasmaPy/PlasmaPy#318]
I propose that we explicitly state typical outputs for our plasma functions in different regimes. This would go in the documentation, examples, and tests for our functions.

We should initially select regimes that we are familiar with. That means having a physical intuition for expected values, or at least, having a table of expected values from a textbook or publication.

For new plasma physics students, this could serve as a tutorial to go test our functions and see if they get the expected values. They can then build an intuition for the different plasma regimes and when things drastically change. For example, the classical Coulomb logarithm breaks down at solid density!!!

For experienced plasma physicists who are new to our library, this would serve as a useful survey to see the differences between our functions and what they may have been using up until now. For example, the factor of `np.sqrt(2)` difference in definitions of thermal velocity.

We should make sure to include each of these regimes in our pytests as a standard for all testing (probably doing this with fixtures, as per @StanczakDominik 's suggestion). This will help us make sure the outputs are tested against something physical, instead of just blindly using the output of our functions as the test (since our functions can be wrong!).

Another approach may be to define a `PlasmaBlob` for each regime and pass the instance to each function of interest. This may involve creating some decorator to grab the relevant plasma parameters and pass them to the function of interest, or just passing the parameters from the instance "manually".

**Comment by lemmatum** [comment]
One useful format for new students would be to make a few IPython/Jupyter notebooks as a tutorial on plasma regimes.

**Comment by lemmatum** [comment]
In the more tutorial style documentation, we should have an IPython notebook that can produce an improved version of this type of plasma regime map:

![wdm_statediagram](https://user-images.githubusercontent.com/28945309/38169029-8f6fa4fe-351b-11e8-9265-8157ba795058.jpg)
